redirects-www.conf

```diff
@@ -330,6 +330,8 @@ rewrite ^/resources/about-fec/reports/gifts-to-foreigners-fy2016-letter.pdf http
 
 # Redirects for /resources/cms-content/documents/
 rewrite ^/resources/cms-content/documents/AllPublicFunds.pdf https://www.fec.gov/resources/cms-content/documents/AllPublicFunds.xls redirect;
+rewrite ^/resources/cms-content/documents/FY2016_AFR.pdf https://www.fec.gov/resources/cms-content/documents/FY2016_AFR.pdf redirect;
+rewrite ^/resources/cms-content/documents/fy_2017_Congressional_budget.pdf https://www.fec.gov/resources/cms-content/documents/fy_2017_Congressional_budget.pdf redirect;
```

Comment:
@kathycarothers These don't look right to me, it seems to be a www redirect that is redirecting back to itself? What's the intent of this redirect?